at_server_status/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: at_server_status
+description: The at_server_status library provides an easy means to check on the status of the @root server as well as the @server for a particular @sign.
+documentation: https://atsign.dev/apidocs
+repository: https://github.com/atsign-foundation/at_libraries
+homepage: https://atsign.dev
+
+version: 0.1.0
+
+environment:
+  sdk: '>=2.8.1 <3.0.0'
+
+dependencies:
+  at_lookup:
+    git:
```

Comment:
dependency has to be in https format in local branch
e.g 

  at_lookup:
    git:
      url: https://github.com/atsign-foundation/at_libraries.git
      path: at_lookup
      ref: dev_env
      version: ^1.0.1
and actual version in master branch
e.g 
at_lookup: ^1.0.1